Ethereum for Dart Developers: MetaMask integration in Flutter applications (#4416)
ethereum/ethereum-org-website: src/content/developers/docs/programming-languages/dart/index.md

  if you already know the basics.
- If you prefer the learn by watching a video, you can watch [Build Your First Blockchain Flutter App](https://www.youtube.com/watch?v=3Eeh3pJ6PeA), which is about an hour long
- If you are impatient, you might prefer [Building a Blockchain Decentralized-app with Flutter and Dart on Ethereum](https://www.youtube.com/watch?v=jaMFEOCq_1s), which is only about twenty minutes

> - [Integrating MetaMask in Flutter application](https://youtu.be/8qzVDje3IWk) is a short video which takes you through every step to the basic integration of MetaMask in your Flutter applications.

Reviewer: ```suggestion
- [Integrating MetaMask in Flutter application](https://youtu.be/8qzVDje3IWk) - this short video takes you through the steps of integrating MetaMask into your Flutter applications.
```
